build.sbt

```diff
@@ -251,7 +251,13 @@ lazy val munitScalacheck = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(
     moduleName := "munit-scalacheck",
     sharedSettings,
-    libraryDependencies += "org.scalacheck" %%% "scalacheck" % "1.15.1"
+    libraryDependencies += {
+      // scalcheck was wrongly released for 1.5.1 instead of 1.15.1
+      if (scalaVersion.value == "3.0.0-M3")
+        "org.scalacheck" %%% "scalacheck" % "1.5.1"
```

Comment:
I highly recommend against doing that, this will create too much pain downstream

Reply:
Fixed, thanks for the help!